generators/client/templates/angular/src/main/webapp/app/core/auth/password-reset.service.ts.ejs

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { SERVER_API_URL } from 'app/app.constants';
+
+@Injectable({ providedIn: 'root' })
+export class PasswordResetService {
+    constructor(private http: HttpClient) {}
+
+    getResetCode(mail: string): Observable<string> {
+        return this.http.post(SERVER_API_URL + 'api/account/reset-password/code', mail, { responseType: 'text' });
+    }
+
+    getMailStatus(): Observable<any> {
```

Comment:
this service shouldn't hold this method. Getting `getMailStatus` shouldn't be a responsibility of password service, please move it to profileInfoService